geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/mapreduce/ConverterInputFormat.scala

```diff
@@ -86,13 +96,25 @@ class ConverterRecordReader extends FileStreamRecordReader with LazyLogging {
       case None    => raw
     }
 
+    import scala.collection.JavaConversions._
+    val featureReader = if (retypedSpec != null) {
+      val retypedSft = SimpleFeatureTypes.createType("retyped", retypedSpec)
```

Comment:
should we keep the original sft name here?

Reply:
I wasn't quite sure if doing so would cause a conflict later. Or if I should concat the two.

Reply:
hmm, i think it would be ok... we usually don't cache anything solely by sft name. alternatively we could store the retyped sft type name in the config like we do with the regular sft - but i think in our query planning it just re-uses the same name.